<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsgForm</class>
 <widget class="QWidget" name="OsgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>533</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterLeftRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="Osg3dView" name="osg3dView">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="OsgTreeWidget" name="osgTreeWidget">
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Mask</string>
        </property>
       </column>
      </widget>
      <widget class="OsgPropertyTable" name="osgPropertyTable">
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Property</string>
        </property>
        <property name="toolTip">
         <string>Name of the Object attribute </string>
        </property>
        <property name="whatsThis">
         <string>Name of the Object attribute </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
        <property name="toolTip">
         <string>The value of the property named in the first column</string>
        </property>
        <property name="whatsThis">
         <string>The value of the property named in the first column</string>
        </property>
       </column>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Camera</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Center</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Eye Pt</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="layersGroupBox">
          <property name="title">
           <string>Layers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OsgTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>OsgTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>OsgPropertyTable</class>
   <extends>QTableWidget</extends>
   <header>OsgPropertyTable.h</header>
  </customwidget>
  <customwidget>
   <class>Osg3dView</class>
   <extends>QOpenGLWidget</extends>
   <header>Osg3dView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
